Makefile

```diff
@@ -7,11 +11,11 @@ build-docker:
 build-kubernetes:
 	docker build -f _infra/docker/Dockerfile .
 
-start:
-	pipenv run python run.py
+load-design-system-templates:
+	pipenv run ./scripts/load_templates.sh $(DESIGN_SYSTEM_VERSION)
 
-UNIT_TESTS=tests/unit
-INTEGRATION_TESTS=tests/integration
+start: load-design-system-templates
```

Comment:
do we need to load templates if its the same version?

Reply:
If the templates are no longer checked into the repo like I mentioned above, these will need to be loaded in each time and doing it this way will mean that a manual copy/run of the load script isn't needed to be done. You can just run `make start`. Also when changing DS versions, ensuring clearing out the templates and pulling them in again will automatically make sure the latest templates are being used and stop any problems such as if templates are renamed in the DS for example, when there could be the potential for both the old and new versions of a template to exist in the repo. I think this is all beneficial and doesn't slow down the `make start` command at all.

Reply:
I get that... for development purposes, it will work but as I said we deploy frontstage as a container images, I don't think curl commands can run while deployments.